source/MRMesh/MRIntersection.h

```diff
@@ -1,21 +1,141 @@
 #pragma once
 
+#include "MRMeshFwd.h"
 #include "MRPlane3.h"
 #include "MRLine3.h"
+#include "MRLineSegm.h"
 #include <optional>
 
 namespace MR
 {
 
+template<typename T>
+const T errorLimit = std::numeric_limits<T>::epsilon() * 20.f;
```

Comment:
better make it not a constant but a parameter of functions below with default value. And "20.f" is a float, so better is simply "20" or T(20)

Reply:
reworked